### PR TITLE
Internal: Editor V2 - Improve test-utils lib

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: packages/packages/documents/src/components/top-bar/settings-button.tsx [hunk 1]
```diff
@@ -17,14 +17,16 @@ export default function SettingsButton( { type }: Props ) {
...
 		<Tooltip title={ title }>
-			<ToggleButton
-				value="document-settings"
-				selected={ isActive }
-				disabled={ isBlocked }
-				onChange={ () => openRoute( 'panel/page-settings/settings' ) }
-			>
-				<CogIcon />
-			</ToggleButton>
+			<span>
```
[the remark this answers is not in the document]
Comment (Nevoss) [reply packages/packages/documents/src/components/top-bar/settings-button.tsx:20]: Done

**Review by StyleShit — DISMISSED** [review]

File: packages/packages/documents/src/components/top-bar/settings-button.tsx [hunk 2]
```diff
@@ -17,15 +17,19 @@ export default function SettingsButton( { type }: Props ) {
...
-				value="document-settings"
-				selected={ isActive }
-				disabled={ isBlocked }
-				onChange={ () => openRoute( 'panel/page-settings/settings' ) }
-				size="small"
-			>
-				<SettingsIcon />
-			</ToggleButton>
+			{ /* @see https://mui.com/material-ui/react-tooltip/#disabled-elements */ }
+			<span aria-label={ undefined }>
```
Comment (rotemee) [inline packages/packages/documents/src/components/top-bar/settings-button.tsx:21]:
```suggestion
			<Box component="span">
```

File: packages/packages/top-bar/src/components/ui/toolbar-menu-item.tsx [hunk 3]
```diff
@@ -8,7 +8,10 @@ type Props = IconButtonProps & {
 export default function ToolbarMenuItem( { title, ...props }: Props ) {
 	return (
 		<Tooltip title={ title }>
-			<IconButton { ...props } aria-label={ title } size="small" />
+			{ /* @see https://mui.com/material-ui/react-tooltip/#disabled-elements */ }
+			<span aria-label={ undefined }>
```
Comment (rotemee) [inline packages/packages/top-bar/src/components/ui/toolbar-menu-item.tsx:12]: Box.

File: packages/packages/top-bar/src/components/ui/toolbar-menu-toggle-item.tsx [hunk 4]
```diff
@@ -17,7 +17,10 @@ export default function ToolbarMenuToggleItem( { title, onClick, ...props }: Pro
 				},
 			} }
 		>
-			<ToggleButton { ...props } onChange={ onClick } size="small" />
+			{ /* @see https://mui.com/material-ui/react-tooltip/#disabled-elements */ }
+			<span aria-label={ undefined }>
```
Comment (rotemee) [inline packages/packages/top-bar/src/components/ui/toolbar-menu-toggle-item.tsx:21]: Box.

**Review by rotemee — CHANGES_REQUESTED** [review]

File: packages/packages/documents/src/components/top-bar/settings-button.tsx (hunk 2 above)
Comment (Nevoss) [reply packages/packages/documents/src/components/top-bar/settings-button.tsx]: Done

File: packages/packages/top-bar/src/components/ui/toolbar-menu-item.tsx (hunk 3 above)
Comment (Nevoss) [reply packages/packages/top-bar/src/components/ui/toolbar-menu-item.tsx]: Done

File: packages/packages/top-bar/src/components/ui/toolbar-menu-toggle-item.tsx (hunk 4 above)
Comment (Nevoss) [reply packages/packages/top-bar/src/components/ui/toolbar-menu-toggle-item.tsx]: Done

**Review by rotemee — APPROVED** [review]

**Pull request merged**